cmd/waku/flags.go

```diff
@@ -551,6 +551,13 @@ var (
 		Destination: &options.RESTServer.RelayCacheCapacity,
 		EnvVars:     []string{"WAKUNODE2_REST_RELAY_CACHE_CAPACITY"},
 	})
+	RESTFilterCacheCapacity = altsrc.NewIntFlag(&cli.IntFlag{
```

Comment:
@NagyZoltanPeter , it might be interesting to add this flag to nwaku, or maybe we should merge both message caches for both filter and relay? what do you think, @harsh-98